cuda/matrix/fbcsr_kernels.cu

```diff
@@ -145,8 +329,21 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
 template <typename ValueType, typename IndexType>
 void is_sorted_by_column_index(
     std::shared_ptr<const CudaExecutor> exec,
-    const matrix::Fbcsr<ValueType, IndexType> *to_check,
-    bool *is_sorted) GKO_NOT_IMPLEMENTED;
+    const matrix::Fbcsr<ValueType, IndexType> *const to_check,
+    bool *const is_sorted)
```

Comment:
*const ?

Reply:
That's just part of my "everything is const by default" philosophy. Prevents reassigning the pointer by mistake inside the function body.